pkg/section/local_module.go

```diff
@@ -59,10 +59,10 @@ func findLocalModules() ([]string, error) {
 
 	uniqueModules := make(map[string]struct{})
 
-	for _, pkg := range packages {
-		if len(pkg.Errors) != 0 {
-			return nil, fmt.Errorf("error reading local packages: %v", pkg.Errors)
-		}
+	for _, pkg := range pkgs {
+		//if len(pkg.Errors) != 0 {
```

Comment:
Could we check it is a no pkg error and still handle other error?

Reply:
Reworked with some additional logic for module name detection.